docs/en/stack/ml/anomaly-detection/ml-ad-job-types.asciidoc

```diff
@@ -0,0 +1,131 @@
+metrics in the job. Multi-metrics jobs provide better results when the 
+influencers are shared across the detectors.
+
+Splitting the analysis by a field enables you to model each value of that field 
+independently. For example, you can split the analysis of your log data set by 
+the `host` field which results in independent baselines for each host (each 
+value of the `host` field) in your data set. If you have a `count` function that 
+detects anomalies in the `error_code` field, and your data is split by the 
+`host` field, then the unusual number of events in the `error_code` field is 
+reported in the context of each host independently. In this case, an observed 
+anomaly in one host does not affect the baseline of another host.
+
+Multi-metric jobs are recommended for complex use cases where you want to detect 
+anomalous behavior in multiple aspects of your data and/or analyze the data in 
+the context of distinct values of a field.
+
+
+[discrete]
+[[population-jobs]]
+## Population jobs
+
+In the case of the population jobs, the analyzed data is split by the distinct 
+values of a field. This field defines what is called a population. The splits 
+are analyzed in the context of all the splits to find unusual values in the 
+population. In other words, the population analysis is a comparison of an 
+individual entity against a collective model of all members in the population as 
+witnessed over time.
+
+For example, if you want to detect IP addresses with unusual request rates 
+compared to the number of requests coming from other IP addresses, you can use a 
+population job. That job has a `count` function to detect unusual number of 
+requests and the analysis is split by the `client_ip` field. In this context, an 
+event is anomalous if the request rate of an IP address is unusually high or low 
+compared to the request rate of all IP addresses in the population. The 
+population job builds a model of the typical number of requests for the IP 
+addresses collectively and compares the behavior of each IP address against that 
+collective model to detect outliers. 
+
+
+[discrete]
+[[advanced-jobs]]
+## Advanced jobs
+
+Advanced jobs give you all the flexibility that's possible in the 
+{ref}/ml-put-job.html[create {anomaly-jobs} API]. At the extreme, you can switch 
+to directly edit the JSON that will be sent to this endpoint. All the other 
+types of jobs described in this page _can_ be created as advanced jobs, but the 
+more specialized wizards make it easier to create jobs for common situations. 
+You can create an advanced job if you are familiar with all the functionality 
+that {ml} {anomaly-detect} provides and want to do something that the more 
+specialized wizards do not allow you to do.
+
+
+[discrete]
+[[categorization-jobs]]
+## Categorization jobs
+
+Categorization jobs cluster similar text values together, classify them into 
+categories and detect anomalies within the categories. Categorization works best 
```

Comment:
Per https://brand.elastic.co/302f66895/p/194a3b-writing-style-guide/t/33fa39 we generally use the serial comma:
```suggestion
categories, and detect anomalies within the categories. Categorization works best 
```